tests/validation_tests/round_four_specific_validations/test_tf_r4_specific_validations.py

```diff
@@ -595,15 +606,22 @@ def test_validate_funding_spent(mocker, allocated_funding):
             sheet="Funding",
             section="Project Funding Profiles - Project 1",
             column="Grand Total",
-            message=msgs.OVERSPEND_PROJECT,
-            row_indexes=[45],
+            message=msgs.OVERSPEND_CDEL,
+            row_indexes=[41],
+        ),
+        TownsFundRoundFourValidationFailure(
+            sheet="Funding",
+            section="Project Funding Profiles - Project 1",
+            column="Grand Total",
+            message=msgs.OVERSPEND_RDEL,
+            row_indexes=[44],
```

Comment:
Can we get a test showing an example of the function not returning any validation errors?

Reply:
Added 👍

Reply:
I can't see the test that does this. What's it called?